swift-sdk/IterableInboxNavigationViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Created by Tapash Majumder on 4/30/19.
+//  Copyright © 2019 Iterable. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+open class IterableInboxNavigationViewController: UINavigationController {
+    /// If you want to use a custom layout for your Inbox TableViewCell
+    /// this is where you should override it. Please note that this assumes
+    /// that the xib is present in the main bundle.
+    @IBInspectable public var cellNibName: String? = nil {
+        didSet {
+            if let inboxViewController = viewControllers[0] as? IterableInboxViewController {
+                inboxViewController.cellNibName = cellNibName
+            }
+        }
+    }
+
+    // MARK: Initializers
+    
+    /// This initializer should be used when initializing from Code.
+    public init() {
+        super.init(rootViewController: IterableInboxViewController(style: .plain))
+        setup()
```

Comment:
Since we're initializing with an existing instance of an `IterableInboxViewController`, won't `setup()` log an error because there is already a view controller in the stack? (line 59)

Reply:
Good catch!